tui/install.go

```diff
@@ -138,3 +164,46 @@ func newInstallPage(tui *Tui) (Page, error) {
 
 	return page, nil
 }
+
+func downloadInstallerMessage(header string, installConf string) {
+	var result Messenger
+
+	downloadURL := installDataURLBase + installConf
+	msgFile, err := conf.FetchRemoteConfigFile(downloadURL)
+	if err != nil {
+		log.Debug("Failed to download the %s message: %s", header, err)
+		return
+	}
+
+	configStr, err := ioutil.ReadFile(msgFile)
+	if err != nil {
+		log.Debug("Failed to read the %s file: %s", header, err)
+		return
+	}
+
+	if err := yaml.Unmarshal(configStr, &result); err != nil {
+		log.Debug("Failed to parse the %s YAML file: %s", header, err)
+		return
+	}
+
+	if result.Messages == nil {
+		log.Debug("%s has no message content", header)
+		return
+	}
+
+	var lines []string
+	found := false
+	for _, message := range result.Messages {
+		line := strings.TrimSpace(message.Default)
+		if len(line) > 0 {
+			// We found at least one line with non-whitespace
+			found = true
```

Comment:
If we are not skipping empty lines why do we need this flag "found"?

Reply:
This is fixed